### DISCUSSION TITLE
Problem of proof in Box 3.1 (Page 55)

### DISCUSSION THREAD

**Discussion by UePG-21** [MathFoundationRL/Book-Mathematical-Foundation-of-Reinforcement-Learning#3] [General]
Hi Professor,

In part 4, isn't $||x^{\ast} - x_n|| \le \frac{\gamma^{n}}{1-\gamma}||x_1 - x_0||$ indicates a linear convergence?

**Comment by MathFoundationRL** [comment]
As n goes to infinity, gamma^n converges to zero exponentially fast.

**Reply by UePG-21** [reply]
Thanks for your quick replay!

But based on my understanding, gamma^n is a power function instead of exponential function of n, since there's even not an "e". Also, this definition may be an endorsement of my statement.

<img width="583" alt="Weixin Image_20231129235717" src="https://github.com/MathFoundationRL/Book-Mathmatical-Foundation-of-Reinforcement-Learning/assets/81507788/ebbd620e-6e32-41e9-92b3-bbb7ef461bc7">

**Comment by MathFoundationRL** [comment]
Convergence is a basic concept in control theory. I am sure that it is exponential, but people may have different definitions.

**Reply by UePG-21** [reply]
Got it, thanks a lot! The course and book are really great works!
